packages/cozy-client/src/CozyLink.js

```diff
@@ -5,6 +5,7 @@ export default class CozyLink {
     }
   }
 
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Arg! Just another reason to move to Typescript I guess... :smile:

Reply:
Here I just put that because I guess the method is not implemented yet. If it will stay in this state we can remove all parameters here.